### PR TITLE
chore: modify some CODEOWNERS

### PR DESCRIPTION
**Pull request by zhongjiajie**
I'll be away from the community for a while, and I've removed me from the codeowner configurations to reduce notification for my mailbox

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -23,7 +23,7 @@
 /dolphinscheduler-registry/ @caishunfeng @ruanwenjun
 /dolphinscheduler-api/ @caishunfeng @SbloodyS
 /dolphinscheduler-dao/ @caishunfeng @SbloodyS
-/dolphinscheduler-dao/src/main/resources/sql/ @zhongjiajie
+/dolphinscheduler-dao/src/main/resources/sql/
```
Comment (zhongjiajie) [inline .github/CODEOWNERS]: did some of our team interested be new code owner for `/dolphinscheduler-dao/src/main/resources/sql/` module?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -36,10 +36,10 @@
...
-/dolphinscheduler-tools/ @caishunfeng @SbloodyS @zhongjiajie @EricGao888
-/script/ @caishunfeng @SbloodyS @zhongjiajie @EricGao888
+/dolphinscheduler-tools/ @caishunfeng @SbloodyS @EricGao888
+/script/ @caishunfeng @SbloodyS @EricGao888
 /dolphinscheduler-ui/ @songjianet @Amy0104
-/docs/ @zhongjiajie @EricGao888
-/licenses/ @zhongjiajie
-/images/ @zhongjiajie @EricGao888
+/docs/ @EricGao888
+/licenses/
```
Comment (zhongjiajie) [inline .github/CODEOWNERS]: did some of our team interested be new code owner for `/licenses/` directory?

**Review by SbloodyS — DISMISSED** [review]
LGTM

**Review by ruanwenjun — DISMISSED** [review]
+1

File: .github/CODEOWNERS (hunk 1 above)
Comment (zhongjiajie) [reply .github/CODEOWNERS]: how about @EricGao888 ?

**Review by caishunfeng — DISMISSED** [review]
+1

File: .github/CODEOWNERS (hunk 1 above)
Comment (EricGao888) [reply .github/CODEOWNERS]: Sure

Comment (zhongjiajie) [reply .github/CODEOWNERS]: will add one more commit and assign those to you 💯

**Review by rickchengx — APPROVED** [review]

**Pull request merged**